components/compliance-service/integration_test/reports_projects_update_test.go

```diff
@@ -1293,35 +1294,37 @@ func TestProjectUpdate(t *testing.T) {
 	for _, test := range cases {
 		t.Run(fmt.Sprintf("report match: %s", test.description),
 			func(t *testing.T) {
-				suite.InsertInspecReports([]*relaxting.ESInSpecReport{test.report})
-				suite.InsertInspecSummaries([]*relaxting.ESInSpecSummary{test.summary})
+				_, err := suite.InsertInspecReports([]*relaxting.ESInSpecReport{test.report})
+				require.NoError(t, err)
+				_, err = suite.InsertInspecSummaries([]*relaxting.ESInSpecSummary{test.summary})
+				require.NoError(t, err)
```

Comment:
In #464 I'm seeing failures from this test when the changes in that branch are seemingly unrelated. Previously this would fail silently.